server/src/main/java/thesistrack/ls1/service/UploadService.java

```diff
@@ -63,7 +68,15 @@ public String store(MultipartFile file) {
     }
 
     public FileSystemResource load(String filename) {
```

Comment:
Please add a check (like above) to stop the client from inputting `..` values breaking out of the `rootLocation`

Reply:
makes sense. Adds a bit more security in case of bugs